seed.js

```diff
@@ -3,69 +3,62 @@
 
 var db = require('./models');
 
-var characters = [
+var characterList = [
 	{
 		name: "Spider-Man",
-		Abilities: ["Web-Slinger", "Swingin'", "Thwip, Thwip, Pow!"], 
+		abilities: ["Web-Slinger", "Swingin'", "Thwip, Thwip, Pow!"], 
 		image: "/images/Spider-Man-Free-Download-PNG.png"
 	}, {
 		name: "Iron-Man",
-		Abilities: ["Repulsor Ray", "Uni-Beam", "Billion-Dollar Punch"], 
+		abilities: ["Repulsor Ray", "Uni-Beam", "Billion-Dollar Punch"], 
 		image: "/images/Iron-Man-PNG-Image.png"
 	}, {
 		name: "Captain-America",
-		Abilities: ["Shield Bash", "Shield Toss", "American Justice"], 
+		abilities: ["Shield Bash", "Shield Toss", "American Justice"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Dead-Pool",
-		Abilities: ["Spinning Groin Punch", "Spinny Sword Attack", "Hold Up"], 
+		abilities: ["Spinning Groin Punch", "Spinny Sword Attack", "Hold Up"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Ultron",
-		Abilities: ["Encephalo-Ray", "Eradication"], 
+		abilities: ["Encephalo-Ray", "Eradication"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"
 	}, {
 		name: "Black-Panther",
-		Abilities: ["Roundhouse Tactical Strikes", "Panther Strike", "For Wakanda!"], 
+		abilities: ["Roundhouse Tactical Strikes", "Panther Strike", "For Wakanda!"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Guillotine",
-		Abilities: ["La Force d'Ancetres", "Coup de Massue", "Coup de Grace"], 
+		abilities: ["La Force d'Ancetres", "Coup de Massue", "Coup de Grace"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Storm",
-		Abilities: ["Lightning Strike", "Gale", "Tempest"], 
+		abilities: ["Lightning Strike", "Gale", "Tempest"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Winter-Soldier",
-		Abilities: ["Deuces are wild", "Fist of the Red Army"], 
+		abilities: ["Deuces are wild", "Fist of the Red Army"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "Groot",
-		Abilities: ["Grow limbs into Earth", "I am Groot"], 
+		abilities: ["Grow limbs into Earth", "I am Groot"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}, {
 		name: "DareDevil",
-		Abilities: ["Stick Strike", "Flash Kick", "Deliverance"], 
+		abilities: ["Stick Strike", "Flash Kick", "Deliverance"], 
 		image: "/images/Doctor-Strange-PNG-Image.png"	
 	}
 ];
 
+db.Character.remove({}, function(err, characters){
+	db.Character.create(characterList, function(err, createdCharacters){
+	  if (err){
+	    console.log("Error:", err);
+	    return;
+	  }
 
-
-// db.Character.remove({}, function(err, characters){
```

Comment:
Your seed.js file should not be commented out. How will you seed your database, if you do not have any code to run that will fill the database with your hard-coded data?